.github/workflows/test-registration1-e2e.yaml

```diff
@@ -89,7 +94,7 @@ jobs:
           E2E_NEW_USER_STORAGE_STATE: ${{ secrets.E2E_NEW_USER_STORAGE_STATE}}
           HAPPO_API_KEY: ${{ secrets.HAPPO_API_KEY }}
           HAPPO_API_SECRET: ${{ secrets.HAPPO_API_SECRET }}
-          HAPPO_NONCE: ${{ github.sha }}-reg1
+          HAPPO_NONCE: ${{github.run_id}}${{ github.run_attempt}}-reg1
```

Comment:
Not sure if this will actually do anything, but I am trying another unique identifier for Happo nonce to see if that solves diffs getting approved affecting other PRs.